spaceR/spaceR/ViewModels/LaunchpadViewModel.swift

```diff
@@ -9,26 +9,44 @@ import Foundation
 
 class LaunchpadViewModel {
     
+    private var repository: LaunchpadRepositoryType?
+    private weak var delegate: ViewModelDelegate?
     private var allLaunchpads: [Launchpads]
     
-    init() {
+    init(repository: LaunchpadRepositoryType,
+         delegate: ViewModelDelegate) {
+        self.repository = repository
+        self.delegate = delegate
         self.allLaunchpads = []
     }
     
-    func setAllLaunchpads(launchpads: [Launchpads]) {
+    func set(launchpads: [Launchpads]) {
```

Comment:
Don't think you need a function for this you can set the variable directly 😄

Reply:
🤙🏼